src/Microsoft.AspNetCore.Server.Kestrel/KestrelServerLimits.cs

```diff
@@ -23,6 +23,9 @@ public class KestrelServerLimits
         // Matches the default LimitRequestFields in Apache httpd.
         private int _maxRequestHeaderCount = 100;
 
+        // Matches the default maxRequestEntityAllowed in IIS.
+        private long _maxRequestBodySize = 200 * 1024;
```

Comment:
+1 for 4 MB as default. 200kB seems too small. One common scenario that usually exceeds 200 kB: image uploads.